src/wolfgame.py

```diff
@@ -3349,7 +3346,7 @@ def begin_day(cli):
     var.DISEASED = set()
     var.MISDIRECTED = set()
     var.DYING = set()
-
+    var.LAST_GOAT = {}
```

Comment:
What is the purpose of setting this variable here back to an empty dict? Also, since the dict already exists, this should be `var.LAST_GOAT.clear()`. That way we keep the same dictionnary for the entire span of the process.